vsphere/vsphere.go

```diff
@@ -629,12 +630,19 @@ func (ops *vsphereOps) Expand(
 
 	task, err := vm.Reconfigure(ctx, spec)
 	if err != nil {
+		if strings.Contains(err.Error(), ".vmdk not found") {
```

Comment:
".vmdk not found" make this a constant variable

Reply:
Done